autoload/vsnip/session/snippet.vim

```diff
@@ -352,6 +352,8 @@ function! s:Snippet.normalize() abort
       endif
     endif
 
+    let self.placeholder = v:null
+    let self.text = v:null
     let self[a:node.type] = {
```

Comment:
Currently, this supports `placeholder` or `text` node but I decided to remove `placeholder normalizing` so please simplify it.

After this improvements, `s:Snippet.normalize` does only merge `adjacent text node`.

Reply:
Thank you. I agree with your opnion.